app/src/main/res/layout/item_news.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <import type="org.schulcloud.mobile.controllers.main.NewsListAdapter.NewsViewHolder" />
+
+        <variable
+            name="news"
+            type="org.schulcloud.mobile.models.news.News" />
+    </data>
+
+    <android.support.v7.widget.CardView
+        android:id="@+id/news_card_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <LinearLayout
```

Comment:
Deep Layouts take more time to render and therefore should be avoided. You can use a ConstraintLayout instead, e.g. https://github.com/schul-cloud/schulcloud-mobile-android/blob/128-readd-files/app/src/main/res/layout/fragment_file_overview.xml#L32